docs/install.md

```diff
@@ -150,7 +150,15 @@ Select the following tabs for the installation description of your target.
     <font size="+2">3. First boot of the new VirtualBox image</font>
 
     Press the start button (green arrow) to 'boot up' your system based on the DietPi image.
-    If you do not have a wired LAN connection you have to change the network settings matching to your environment (files `\boot\dietpi.txt` and `\boot\dietpi-wifi.txt`).
+
+    <font size="+1">3.1 Configuration hints: IPv6</font>
+
+    In some cases the system does not find the update servers (e.g. at the very first update run). This is signaled during the "apt update" procedure of the first boot startup.  
+    To overcome this, open a subshell (or an additional ssh window), start `dietpi-config` and disable **IPv6** within the Network options.
+
+    ![VirtualBox virtual machine IPv6 deactivate screenshot](assets/images/dietpi-VirtualBox-IPv6.png){: width="500" height="225" loading="lazy"}
+
+    Then exit `dietpi-config`and reboot. After this the first time installer procedure should run again from the start.
```

Comment:
@MichaIng: Do you think this is an effect of the combination "Virtual machine" (every: VBox, WMware, Hyper-V) and WiFi on the host and a bridged network?

Then I would add it to every VM section, because we have not combined VM installation section at the end (precise: We have it, but we do not have a fitting heading left).

Reply:
Yes indeed it is a general expected issue: https://communities.vmware.com/t5/VMware-Fusion-Discussions/IPv6-Bridged-Wireless/td-p/2038235

Okay I agree, let's add this info to all VMs: When the host is connected with WiFi, IPv6 needs to be disabled, since IPv6 is not able to work with WiFi bridges.

Reply:
Ok, I will do this.